### PR TITLE
Pass on the documentation of std.contract

### PR DESCRIPTION
**Pull request by yannham**
Depends on #1263.

### CODE REVIEW

File: stdlib/std.ncl [hunk 1]
```diff
@@ -909,8 +909,8 @@
 
     label
       | doc m%"
-          The label submodule gathers functions that manipulate the label of
-          a contract.
+          The label submodule provides functions that manipulate the label of a
+          contract.
```
Comment (vkleen) [inline stdlib/std.ncl:913]: I'm not sure what else to write here, but I feel like a label doesn't really belong to a contract, but to a contract application.

**Review by vkleen — APPROVED** [review]

File: stdlib/std.ncl (hunk 1 above)
Comment (yannham) [reply stdlib/std.ncl:913]: Ah, you're right indeed. I guess saying exactly that is fine.

Comment (yannham) [inline stdlib/std.ncl:913]:
```suggestion
          The label submodule provides functions that manipulate the label
          associated to a contract application.
```

Comment (yannham) [reply stdlib/std.ncl:913]: @vkleen do you think that'd be ok?

Comment (vkleen) [reply stdlib/std.ncl:913]: I'd say "associated with" (because otherwise my thesis adviser will come haunt me) and "manipulates". Otherwise, looks good 👍

**Pull request merged**